src/vcProject.cpp

```diff
@@ -16,8 +16,8 @@ const char *vcProject_ErrorToString(vdkError error)
     return vcString::Get("errorInvalidParameter");
   case vE_OpenFailure:
     return vcString::Get("errorOpenFailure");
-  case vE_NotSupported:
-    return vcString::Get("errorUnsupported");
+  case vE_WriteFailure:
+    return vcString::Get("errorFileExists");
```

Comment:
You probably shouldn't remove the `vE_NotSupported` message?

Reply:
It is not used now.